frontend/shared/components/src/periods/EditRegistrationPeriodView.vue

```diff
@@ -110,6 +155,31 @@ const endDate = computed({
     set: endDate => addPatch({ endDate }),
 });
 
+watch([startDate, endDate], validate);
```

Comment:
Watchers voor validatie lijkt mij echt foutgevoelig, zou ik echt niet doen. Dan moet je blur events gaan checken en met delays werken etc, lijkt me het niet waard. Op 'save' is meer dan voldoende, en niet irritant voor mensen.